vue-client/src/components/inspector/search-window.vue

```diff
@@ -131,7 +131,11 @@ export default {
             this.$nextTick(() => {
               this.resetSearch();
               if (this.itemInfo !== null) {
-                const cleanedChipString = DisplayUtil.getItemLabel(this.itemInfo, this.resources, this.inspector.data.quoted, this.settings).replace(/#|_|•|\[|\]/g, ' ').replace(/  +/g, ' ');
+                const cleanedChipString = DisplayUtil.getItemLabel(this.itemInfo, this.resources, this.inspector.data.quoted, this.settings)
+                  .replace(/#|_|•|\[|\]/g, ' ')
+                  .replace(/  +/g, ' ')
+                  .replace(new RegExp(`s?({(.*?)(${this.settings.availableUserSettings.languages.map(lang => StringUtil.getUiPhraseByLang('without', lang.value, this.resources.i18n)).join('|')})(.*?)})`, 'g'), '')
```

Comment:
This will work fine in this case,  as long as the word without isn't translated into something with regexp-specific characters (which I really don't expect here).

We should in general be very careful when creating regexps like this though, since they could blow up (e.g. if they contain unbalanced parens). See [MDN docs on regexp escaping](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Guide/Regular_Expressions#escaping) for details.

Reply:
Fix using lo-dash’s `escapeRegExp` can be found here: https://github.com/libris/lxlviewer/pull/877
(not sure if a pull request is needed for such a small change)